src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -78,6 +114,43 @@ protected override void BeginProcessing()
         /// </summary>
         protected override void ProcessRecord()
         {
+            if ( ParameterSetName == StringHashParameterSet)
+            {
+                if (InputString == null)
+                {
+                    WriteStringHashInfo(Algorithm, null, null, Encoding);
+                }
+                else
+                {
+                    foreach (string str in InputString)
+                    {
+                        try
+                        {
+                            //if (str == null)
```

Comment:
Yes,  `str == null`  is the only error condition.
It seems `if` is lighter code then `try-catch`.

Fixed.